playground/playground.gd

```diff
@@ -95,39 +93,39 @@ func _on_capture_log_button_pressed() -> void:
 			level = "error"
 	var message = $"%LogMessage".text
 	msg("[i]discord.command_capture_log(\"" + level + "\", \"" + message + "\")[/i]")
-	var _result = await discord.command_capture_log(level, message)
+	var _result := await discord.command_capture_log(level, message)
 	msg("[i][b]no output[/b][/i]")
 
 
 func _on_encourage_hw_accel_pressed() -> void:
 	msg("[i]discord.command_encourage_hardware_acceleration()[/i]")
-	var result = await discord.command_encourage_hardware_acceleration()
+	var result := await discord.command_encourage_hardware_acceleration()
 	msg("Hardware acceleration is: " + ("Enabled" if result["enabled"] else "Disabled"))
 
 
 func _on_external_url_button_pressed() -> void:
-	var url = $"%ExternalUrlInput".text
+	var url: String = $"%ExternalUrlInput".text
 	msg("[i]discord.command_open_external_link(" + str(url) + ")[/i]")
-	var result = await discord.command_open_external_link(url)
+	var result := await discord.command_open_external_link(url)
 	if (result["opened"]):
 		msg("URL opened")
 	else:
 		msg("URL did not open")
 
 
 func _on_set_orientation_lock_button_pressed() -> void:
-	var lock_state = -1 if $"%LockState".get_selected_id() == 0 else $"%LockState".get_selected_id()
-	var pip_lock_state = -1 if $"%PipLockState".get_selected_id() == 0 else $"%PipLockState".get_selected_id()
-	var grid_lock_state = -1 if $"%GridLockState".get_selected_id() == 0 else $"%GridLockState".get_selected_id()
+	var lock_state: int = -1 if $"%LockState".get_selected_id() == 0 else $"%LockState".get_selected_id()
+	var pip_lock_state: int = -1 if $"%PipLockState".get_selected_id() == 0 else $"%PipLockState".get_selected_id()
+	var grid_lock_state: int = -1 if $"%GridLockState".get_selected_id() == 0 else $"%GridLockState".get_selected_id()
 
 	msg("[i]discord.command_set_orientation_lock_state(" + str(lock_state) + ", " + str(pip_lock_state) + ", " + str(grid_lock_state) + ")[/i]")
-	var _result = await discord.command_set_orientation_lock_state(lock_state, pip_lock_state, grid_lock_state)
+	var _result := await discord.command_set_orientation_lock_state(lock_state, pip_lock_state, grid_lock_state)
 	msg("[i][b]no output[/b][/i]")
 
 
 func _on_log_channel_info_button_pressed() -> void:
 	msg("[i]discord.command_get_channel(" + str(discord.channel_id) + ")[/i]")
-	var result = await discord.command_get_channel(discord.channel_id)
+	var result := await discord.command_get_channel(discord.channel_id)
 	msg("[Channel Info] =====================================================")
 	msg("Id: " + str(result["id"]))
```

Comment:
Can't add a suggestion block here, but same as others, property access instead of indexing